waspc/src/Wasp/Generator/WebAppGenerator/AuthG.hs

```diff
@@ -63,20 +63,38 @@ genAuthForms auth =
     [ genLoginForm auth,
       genSignupForm auth
     ]
+    <++> (if AS.App.Auth.isExternalAuthEnabled auth then genExternalAuth auth else return [])
```

Comment:
Same thing as in the other comments, should we move the check into `genExtrernalAuth` for consistency purposes?